advisories/github-reviewed/2022/03/GHSA-3gx9-37ww-9qw6/GHSA-3gx9-37ww-9qw6.json

```diff
@@ -1,17 +1,17 @@
 {
   "schema_version": "1.2.0",
   "id": "GHSA-3gx9-37ww-9qw6",
-  "modified": "2022-03-13T00:53:36Z",
+  "modified": "2022-03-13T00:55:19Z",
   "published": "2022-03-04T00:00:15Z",
   "aliases": [
     "CVE-2022-22947"
   ],
   "summary": "Code injection in Spring Cloud Gateway",
-  "details": "In Spring Cloud Gateway versions prior to 3.1.1+ and 3.0.7+ , applications are vulnerable to a code injection attack when the Gateway Actuator endpoint is enabled, exposed, and unsecured. A remote attacker could make a maliciously crafted request resulting in arbitrary remote execution on the remote host.",
+  "details": "In Spring Cloud Gateway versions prior to 3.1.1+ and 3.0.7+, applications are vulnerable to a code injection attack when the Gateway Actuator endpoint is enabled, exposed, and unsecured. A remote attacker could make a maliciously crafted request resulting in arbitrary remote execution on the remote host.",
   "severity": [
     {
       "type": "CVSS_V3",
-      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H"
+      "score": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:H/A:H"
```

Comment:
The reason for this change is that the details specifically say applications are vulnerable when the endpoint is "enabled, exposed, and unsecured". Whether or not this is the case is beyond the attacker's control.

Reply:
jbmagination-GHSA-3gx9-37ww-9qw6